doc/api/inspector.md

```diff
@@ -121,9 +121,15 @@ session.on('Debugger.paused', ({ params }) => {
 added: v8.0.0
 -->
 
-Connects a session to the inspector back-end. An exception will be thrown
-if there is already a connected session established either through the API or by
-a front-end connected to the Inspector WebSocket port.
+Connects a session to the inspector back-end.
+
+### session.connectToMainThread()
```

Comment:
To clarify again: This *does* connect to the *parent* thread and not the main thread, right? In that case the naming should reflect that, imo

Reply:
All workers will connect to a main thread, I added a test case. This is based on https://github.com/nodejs/node/pull/28872 that makes all workers be reported by the main thread.